checkout-ui-custom/checkout6-custom.js

```diff
@@ -25,27 +25,47 @@ const MAX_TIME_EXPIRATION = 1000 * 60 * 5 // 5 minutes
 !(function () {
   console.log('B2B Checkout Settings')
   let checkVtex = null
-  const translation = {
-    'pt-BR': {
-      cartPurchaseOrderLabel: 'Número do Pedido',
-      createQuoteButtonLabel: 'Criar uma Cotação',
-      messages: {
-        'b2b-access-denied': {
-          title: 'Acesso',
-          detail: 'Você não tem acesso ao checkout',
+
+  const getTranslation = function () {
+    const locale = window.vtex.i18n.getLocale()
+    const translation = {
+      'pt-BR': {
+        cartPurchaseOrderLabel: 'Número do Pedido',
+        createQuoteButtonLabel: 'Criar uma Cotação',
+        messages: {
+          'b2b-access-denied': {
+            title: 'Acesso',
+            detail: 'Você não tem acesso ao checkout',
+          },
+        },
+        clearCartLabel: 'Limpar carrinho',
+      },
+      en: {
+        cartPurchaseOrderLabel: 'Reference or PO Number',
+        createQuoteButtonLabel: 'Create a Quote',
+        messages: {
+          'b2b-access-denied': {
+            title: 'ACCESS',
+            detail: "You don't have access to the checkout",
+          },
         },
+        clearCartLabel: 'Clear Cart',
       },
-    },
-    en: {
-      cartPurchaseOrderLabel: 'Reference or PO Number',
-      createQuoteButtonLabel: 'Create a Quote',
-      messages: {
-        'b2b-access-denied': {
-          title: 'ACCESS',
-          detail: "You don't have access to the checkout",
+      es: {
+        cartPurchaseOrderLabel: 'Número de referencia o número del pedido',
+        createQuoteButtonLabel: 'Crear un presupuesto',
+        messages: {
+          'b2b-access-denied': {
+            title: 'ACCESO',
+            detail: 'No tienes acceso al checkout',
+          },
         },
+        clearCartLabel: 'Limpiar carrito',
       },
-    },
+    }
+
+    // default to EN if language not supported
+    return translation[locale] ? translation[locale] : translation.en
```

Comment:
You can simplify with a nullish coalescing operator

Reply:
Done. I got a linting error using `??` so I went with the logical OR operator instead.